aip/general/0192.md

```diff
@@ -108,6 +105,14 @@ colloquial use that avoiding it would obscure the reference (example: [IBM][]).
 Comments **should** spell and capitalize trademarked names consistent with the
 trademark owner's current branding.
 
+### Deprecations
```

Comment:
All the changes above this line seem irrelevant to this PR. At the very least, the changes at the top of the file need to be reverted so the metadata is handled properly.

Reply:
Done.